pynamodb/attributes.py

```diff
@@ -362,12 +362,9 @@ def serialize(self, value):
 
     def deserialize(self, value):
         """
-        Returns a decoded string from base64
+        Returns a decoded byte string from a base64 encoded value
         """
-        try:
-            return b64decode(value.decode(DEFAULT_ENCODING))
-        except AttributeError:
-            return b64decode(value)
+        return b64decode(value)
```

Comment:
This is a result of being py3 only now?

Reply:
yes, in py3 b64decode takes either a string or bytes so we don't have to try to decode first:
https://docs.python.org/3/library/base64.html#base64.b64decode